typescript/lib/parsers/openai.ts

```diff
@@ -601,12 +607,19 @@ export class OpenAIChatModelParser extends ParameterizedModelParser<Chat.ChatCom
             /*accumulatedData*/ message,
             /*index*/ choice.index
           );
+          if (message === undefined) {
+            continue;
+          }
+
+          const outputData: OutputDataWithValue | string | undefined =
+            buildOutputData(message);
+          if (outputData === undefined) {
+            continue;
+          }
```

Comment:
Same as above

Reply:
Same comment, intentionally can't be null, only `OutputDataWithValue | string | undefined`